File: CourierAmexClient/src/app/features/general/models/country.model.ts
import { BaseEntity } from "@app/models";

export interface CountryModel extends BaseEntity<number> {
  name: string;
  shortname?: string;
  code?: number;
  address?: string;
  notes?: string;
}

export const newCountryModel = {
  id: 0,
  name: '',
  status: 2 //DEFAULT_STATUS_ACTIVE
};

File: CourierAmexAPI/CourierAmex/Models/CountryModel.cs
namespace CourierAmex.Models
{
    public class CountryModel : BaseEntityModel<int>
    {
        public required string Name { get; set; }
        public string? Shortname { get; set; }
        public int Code { get; set; }
        public string? Notes { get; set; }
        public string? Address { get; set; }
    }
}
